README.md

```diff
@@ -197,8 +197,16 @@ cluster. If one is not available, you may deploy this with:
 juju config kubernetes-worker ingress=true
 ```
 
-Ingress resources are DNS mappings to your containers, routed through
-[endpoints](http://kubernetes.io/docs/user-guide/services/)
+In any case, in order to use the ingress, you will need to expose the worker service via
+
+```
+juju expose kubernetes-worker
```

Comment:
This was added with the last update. ~ line number  187. It looks weird having this in 2 places under the same subheading.